geode-docs/developing/query_select/the_select_statement.html.md.erb

```diff
@@ -122,6 +122,39 @@ SELECT DISTINCT * FROM /exampleRegion WHERE ID < 101 ORDER BY ID desc
 **Note:**
 If you are using ORDER BY queries, you must implement the equals and hashCode methods for the objects that you query.
 
+## <a id="select_groupBy" class="no-quick-link"></a>GROUP BY
+
+You can collect data across multiple entries and group the results by one or more columns through the usage of the GROUP BY statement.
+It's important to note some facts about the usage of the statement:
+
+* It can group results by one or more fields.
+* It returns a single record / entry per group.
```

Comment:
The query from the example will actually fail, basically because you're grouping by `region` and that particular field is not part of the projection list (`SELECT` only contains `status`).
Let' say you have a region with the following entries:

```
ID     | Status
1      | active
2      | inactive
3      | active
4      | inactive
```

If you execute the query `SELECT pf.status AS st, MIN(pf.ID) AS mn, MAX(pf.ID) AS mx FROM /myRegion pf GROUP BY pf.status` then the query engine will return 2 results: 1 per group (we have 2 groups here: aka two values for the filed `status`, the one used for the aggregation):

```
st        | mn | mx
active    | 1  | 3
inactive  | 2  | 4
```

Regarding more examples, there's a separate ticket created for that already: [GEODE-6907](https://issues.apache.org/jira/browse/GEODE-6907).